monitoring/monitoring_internal_test.go

```diff
@@ -255,9 +235,15 @@ func TestServices(t *testing.T) {
 
```

Comment:
TestServices should be renamed to TestInstances

Reply:
Done